scripts/provisioning-terraform.sh

```diff
@@ -238,11 +249,11 @@ fi
 mkdir -p "$(pwd)"/tmp
 
 run_containerized_terraform "${GOOGLE_APPLICATION_CREDENTIALS_PATH}" "${VIAI_CAMERA_INTEGRATION_DIRECTORY_PATH}" version
-run_containerized_terraform "${GOOGLE_APPLICATION_CREDENTIALS_PATH}" "${VIAI_CAMERA_INTEGRATION_DIRECTORY_PATH}" init
+run_containerized_terraform "${GOOGLE_APPLICATION_CREDENTIALS_PATH}" "${VIAI_CAMERA_INTEGRATION_DIRECTORY_PATH}" init -backend-config="bucket=tf-state-${DEFAULT_PROJECT}"
```

Comment:
Where do you generate the backend configuration file?

Reply:
[backend.tf](https://github.com/GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration/blob/main/terraform/backend.tf), it was using local, switch to gcs.